sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -560,10 +561,11 @@ def test_loadings_converges():
 
     cca = CCA(n_components=10, max_iter=500)
 
-    with pytest.warns(None) as record:
+    with warnings.catch_warnings():
+        # ConvergenceWarning should not be raised
```

Comment:
Same reason from my review. The `simplefilter` makes it clear that we are testing for `ConvergenceWarning`.

```suggestion
```